ts/utils/share.ts

```diff
@@ -42,8 +43,13 @@ export const isShareEnabled = () =>
 export const saveImageToGallery = (uri: string): TaskEither<Error, string> => {
   const hasPermission = tryCatch(
     () =>
-      hasAndroidPermission(
-        PermissionsAndroid.PERMISSIONS.WRITE_EXTERNAL_STORAGE
+      requestIOAndroidPermission(
+        PermissionsAndroid.PERMISSIONS.WRITE_EXTERNAL_STORAGE,
+        {
+          title: I18n.t("permissionRationale.storage.title"),
+          message: I18n.t("permissionRationale.storage.message"),
+          buttonPositive: I18n.t("global.buttons.choose")
```

Comment:
It is just a check: I expect `ok` in place of `scegli`
Is it what we want?

Reply:
Yes this is intended by design.

Reply:
legal design aka @CaraDune :D